src/app/store/radioStore.ts

```diff
@@ -73,12 +82,18 @@ const useRadioState = create<RadioState>((set) => ({
       );
       return;
     }
+
+    const [station, position, subPosition] = getCallsignParts(callsign);
```

Comment:
Calculate the sub-parts of the callsign so they can be stored with the radio when the radio is created.